src/docfx/lib/log/Telemetry.cs

```diff
@@ -147,5 +184,50 @@ private static string CoalesceEmpty(string? str)
         {
             return string.IsNullOrEmpty(str) ? "<null>" : str;
         }
+
+        private static string SerializeConfigObjectWithMaxLength(JObject graph)
+        {
+            TryRemoveNestedObject(graph);
+            var result = JsonUtility.Serialize(graph);
+            if (result.Length <= MaxEventPropertyLength)
+            {
+                return result;
+            }
+
+            TryRemoveLongArray(graph);
```

Comment:
Feel a bit strange to trim again here and also why not trim JObject as well, how about reduce `MaxChildrenLength` instead?

Reply:
remove it then